### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brunoagretti**
feature: add images to README.md

### CODE REVIEW

**Comment by brunoagretti** [comment]
Please review the changes

File: README.md [hunk 1]
```diff
@@ -11,7 +11,6 @@ El modelado en Protegè se encuentra el archivo X, luego para los datos utilizad
 ## Video con demostración del modelado en Protegè
 Durante la realización de la primer etapa subimos [este video](https://www.youtube.com/watch?v=Af2zAfW4mm0&t=202s&ab_channel=BrunoAgretti) con la intención de explicar a grandes rasgos los problemas encontrados y la solución propuesta.
 
-
```
Comment (JuanWigg) [inline README.md:14]: I believe this space is very important to mantain a well understanding of the README.md. Please put it back before merging

Comment (JuanWigg) [reply README.md:14]: :trollface:

Comment (brunoagretti) [reply README.md:14]: I'm not sure about that. @AgusIGarcia What do you think about that?

Comment (AgusIGarcia) [reply README.md:14]: LGTM, Merge and delete branch plz

Comment (JuanWigg) [reply README.md:14]: Lets gamble, test merging?

**Review by JuanWigg — APPROVED** [review]
RIP Production

**Pull request merged**